tt_metal/impl/buffers/buffer.hpp

```diff
@@ -145,79 +137,48 @@ struct BufferPageMapping {
     std::vector<std::array<uint32_t, 2>> core_shard_shape_;
 };
 
+struct AllocBufferMetadata;
+void EnqueueAllocateBufferImpl(AllocBufferMetadata alloc_md);
+
```

Comment:
Why is this needed here? Doesn't seem like anything in buffer uses it?

Reply:
Good catch, removed